zip-0227.rst

```diff
@@ -118,20 +118,26 @@ This allows the issuer to use the same wallet it usually uses to transfer Assets
 Asset Identifier
 ----------------
 
-For every new Asset, there must be a new and unique Asset Identifier. We define this to be a globally unique pair :math:`(\mathsf{ik}, \mathsf{asset\_desc})`, where :math:`\mathsf{ik}` is the issuance key and :math:`\mathsf{asset\_desc}` is a byte string.
+Every asset must have a unique Asset Identifier. We define this to be a globally unique pair:
+:math:`(\mathsf{ik}, \mathsf{asset\_desc})`, where: :math:`\mathsf{ik}` is the issuance public key and :math:`\mathsf{asset\_desc}` is a UTF-8 byte string.
 
-A given Asset Identifier is used across all Zcash protocols that support ZSAs -- that is, the Orchard-based ZSA protocol and potentially future Zcash shielded protocols. For this Asset Identifier, we derive an Asset Digest, :math:`\mathsf{AssetDigest}`, which is simply is a :math:`\mathsf{BLAKE2b-512}` hash of the Asset Identifier.
-From the Asset Digest, we derive a specific Asset Base within each such shielded protocol (for example :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}_{\mathsf{AssetId}}` for the Orchard-based ZSA protocol), using the applicable hash-to-curve algorithm. This Asset Base is included in shielded notes.
+A given Asset Identifier is used across all Zcash protocols that support ZSAs.
+For this Asset Identifier, we derive an Asset Digest, :math:`\mathsf{AssetDigest}`, which is :math:`\mathsf{BLAKE2b-256}` hash of the Asset Identifier.
+Then, a specific Asset Base is derived from the Asset Digest within each shielded protocol.
+
+For Orchard: 
+:math:`\mathsf{AssetBase}^{\mathsf{Orchard}}_{\mathsf{AssetId}}` , using the applicable hash-to-curve algorithm. This Asset Base is included in shielded notes.
```

Comment:
Also, we should mention that AssetBase for native assets will retain the same personalization as defined in orchard: "z.cash:Orchard-cv"
![image](https://user-images.githubusercontent.com/3682187/218298814-76d8a020-58a9-4f5a-8378-92f3719d1734.png)

Reply:
```rust
/// SWU hash-to-curve personalization for the ZSA asset base generator
pub const ZSA_ASSET_BASE_PERSONALIZATION: &str = "z.cash:OrchardZSA";

Reply:
I will mention this lower down with the `\mathsf{ZSAValueBase^{Protocol}}` definition (approximately line 142)